playground/infrastructure/helper.py

```diff
@@ -44,29 +58,194 @@ def find_examples(work_dir: str) -> List[Example]:
         categories:
             - category-1
             - category-2
+    If some example contain beam tag with incorrect format raise an error.
 
     Args:
         work_dir: directory where to search examples.
+        categories_path: path to the file with all supported categories.
 
     Returns:
         List of Examples.
     """
-    # TODO [BEAM-13135] Implement
-    pass
+    failed = False
+    examples = []
+    supported_categories = _get_supported_categories(categories_path)
+    for root, _, files in os.walk(work_dir):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            extension = filepath.split(os.extsep)[-1]
+            if extension in Config.SUPPORTED_SDK:
+                tag = get_tag(filepath)
+                if tag:
+                    if _validate(tag, supported_categories) is False:
+                        logging.error(filepath + "contains beam playground tag with incorrect format")
```

Comment:
method `_validate` log all errors associated with the tag so it would be more detailed for the user. This log just shows that the file by `filepath` contains an incorrect tag.